config/notifiers.yml

```diff
@@ -0,0 +1,3 @@
+notifiers:
+  default: "https://hooks.slack.com/services/SOMECHANNEL"
```

Comment:
What I mentioned the other day relates to this. Slack's webhooks seem to work for all channels (from the one I tested), so it probably isn't necessary to have separate webhook URLs. We can then simplify this by passing in the webhook URL by an env or similar